distro_test.go

```diff
@@ -163,6 +177,23 @@ configuration: error in GetConfiguration: failed to convert %q to UTF16
 	}
 }
 
+func TestGUID(t *testing.T) {
+	// This test validates that the GUID is properly printed.
+	// Note that windows.GUID has a String method printing the expected
+	// format "{XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX}", but syscall.GUID
+	// does not have such method and prints its contents like any other
+	// struct.
+	distro := wsl.NewDistro(uniqueDistroName(t))
+	err := distro.Register(emptyRootFs)
+	require.NoError(t, err, "could not register empty distro")
+
+	guid, err := distro.GUID()
```

Comment:
Any test when for GUID() itself, when we can make it returning an error? I think a small table test and tricking it here could be useful.

Reply:
Good point. Done.